lib/govuk_publishing_components/presenters/attachment.rb

```diff
@@ -4,12 +4,7 @@ class Attachment
       # DfE, DWP, DHSC, HMRC, DVSA and PHE are taking part in a pilot to use a form
```

Comment:
Should this comment be changed to reflect the current reality?

Reply:
thank you! updated it